modules/determining-upgrade-viability-cv-conditiontype.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="understanding-clusterversion-conditiontypes_{context}"]
+= Understanding cluster version condition types
+ 
+The Cluster Version Operator (CVO) is responsible for collecting the status conditions of the cluster version and its Operators. This status includes the condition type, informing you of the health and current state of the {product-title} cluster.
```

Comment:
s/informing/which informs   Or other verb (tells? which describes the health?). I was a bit confused, thinking this was going to be a list of things.

Reply:
good catch - updated :)